monitor.py

```diff
@@ -210,7 +212,7 @@ def check_job_status(spy_link):
     Returns:
         string: Job run Status 
     '''
-    job_status_url = PROW_VIEW_URL + spy_link[8:] + '/finished.json'
+    job_status_url = prow_view_url + spy_link[8:] + '/finished.json'
```

Comment:
```suggestion
    job_status_url = constants.PROW_VIEW_URL + spy_link[8:] + '/finished.json'
```
Applies for all the other usages